doc/classes/@GlobalScope.xml

```diff
@@ -72,6 +72,14 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="angle_difference">
+			<return type="float" />
+			<param index="0" name="from" type="float" />
+			<param index="1" name="to" type="float" />
+			<description>
+				Returns the difference between the two angles, correctly wrapping around [constant @GDScript.TAU].
```

Comment:
```suggestion
				Returns the difference between the two angles. in the range of [code][-PI, +PI][/code]. When [param from] and [param to] are opposite, returns [code]-PI[/code] if [param from] is smaller than [param to], or [code]PI[/code] otherwise.
```